src/Spectre.libClassifier/OpenCvDataset.cpp

```diff
@@ -48,6 +48,30 @@ OpenCvDataset::OpenCvDataset(OpenCvDataset &&other) noexcept
     other.m_observations.clear();
 }
 
+OpenCvDataset::OpenCvDataset(OpenCvDataset &&first, OpenCvDataset &&second) noexcept
```

Comment:
Do we need this to be a constructor? I mean, `OpenCvDataset` is still growing. Can't we just extract this out of here without performance loss?